### ISSUE TITLE
Add input field for request URL in save request modal

### ISSUE THREAD

**Issue by kartikjain26** [a85/POSTMan-Chrome-Extension#7027]
![Screenshot 2019-07-30 at 2 59 35 PM](https://user-images.githubusercontent.com/26088676/62154574-9b776b00-b324-11e9-8693-957999ac638b.png)

Current save request modal does not support adding request URL.
This looks odd in cases where, a user has an empty collection, clicks on Add Requests and has to save a request with no URL at the beginning.

**Comment by shamasis** [comment]
Yeah. URL sounds useful here, but so does request method and other stuff. I guess we will need to solve it using better in-place editing and keyboard navigation to finally get rid of this modal. 

Thoughts @vkaegis ?
